datasets/acute-care/copy_slivar_with_vep_split.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -ex
+
+gcloud auth configure-docker australia-southeast1-docker.pkg.dev
+skopeo copy docker://docker.io/mwellandcpg/slivar:vep_split docker://australia-southeast1-docker.pkg.dev/cpg-common/images/slivar:vep_split
```

Comment:
Can we move the Docker file into a CPG repository and push the image using CI, instead of copying this "outside" build?